app/usecases/discord.py

```diff
@@ -159,12 +159,8 @@ async def post(self) -> None:
         """Post the webhook in JSON format."""
 
         res: Optional[dict] = None
-        async with app.state.services.http.post(self.url, data=self.json) as req:
-            if req and req.status == 200:
-                res = await req.json()
-
-        if res:
-            logging.debug(f"Webhook response: {res}")
+        async with app.state.services.http.post(self.url, json=self.json) as req:
+            ...
```

Comment:
Would probably be nice to log if an issue with posting them is occuring

Reply:
k